packages/ember-runtime/lib/mixins/enumerable.js

```diff
@@ -872,6 +872,12 @@ const Enumerable = Mixin.create({
     @private
   */
   addEnumerableObserver(target, opts) {
+    deprecate(
+      'Usage of `Enumerable.prototype.addEnumerableObserver` is deprecated.',
+      false,
+      { until: '2.12.0', id: 'ember-metal.enumerable.addEnumerableObserver' }
```

Comment:
I can never remember; does 2.12 mean 2.12 is the last version where this continues to work? I'll have to update my patch accordingly.